pkg/ebpftracer/tracer_decode.go

```diff
@@ -97,18 +55,113 @@ func (t *Tracer) decodeAndExportEvent(ctx context.Context, ebpfMsgDecoder *decod
 		return fmt.Errorf("cannot get container for cgroup %d: %w", eventCtx.CgroupID, err)
 	}
 
+	rawEventTime := eventCtx.Ts
 	eventCtx.Ts = t.bootTime + eventCtx.Ts
 	event := &types.Event{
 		Context:   &eventCtx,
 		Container: container,
 		Args:      parsedArgs,
 	}
 
+	switch eventId {
+	case events.SchedProcessExec:
+		if eventCtx.Pid == 1 {
+			t.cfg.MountNamespacePIDStore.ForceAddToBucket(proc.NamespaceID(eventCtx.MntID), eventCtx.NodeHostPid)
+		} else {
+			t.cfg.MountNamespacePIDStore.AddToBucket(proc.NamespaceID(eventCtx.MntID), eventCtx.NodeHostPid)
+		}
+
+		parentStartTime := time.Duration(0)
+		if eventCtx.Ppid != 0 {
+			// We only set the parent start time, if we know the parent PID comes from the same NS.
+			parentStartTime = time.Duration(eventCtx.ParentStartTime) * time.Nanosecond
+		}
+		execArgs, ok := parsedArgs.(types.SchedProcessExecArgs)
+		if !ok {
+			t.log.Errorf("expected types.SchedProcessExecArgs, but got: %t", parsedArgs)
+			return nil
+		}
+		processStartTime := time.Duration(eventCtx.StartTime) * time.Nanosecond
+
+		t.cfg.ProcessTreeCollector.ProcessStarted(
+			system.GetBootTime().Add(time.Duration(rawEventTime)),
```

Comment:
We have variable t.bootTime, can it be used here instead of calling GetBootTime?

Reply:
I was thinking about this (I changed the signature like 20 times 😅). The reason I went with time is, that the number alone makes it quite confusing what it represents exactly (is it seconds, milliseconds, nanoseconds, and so on).